docs/markdown/Cross-compilation.md

```diff
@@ -35,14 +35,24 @@ three machines are the same. Simple so far.
 Let's next look at the most common cross-compilation setup. Let's
 suppose you are on a 64 bit OSX machine and you are cross compiling a
 binary that will run on a 32 bit ARM Linux board. In this case your
-*build machine* is 64 bit OSX and both your *host* and *target
-machines* are 32 bit ARM Linux. This should be quite understandable as
-well.
-
-It gets a bit trickier when we think about how the cross compiler was
-generated. It was built and it runs on a specific platform but the
-output it generates is for a different platform. In this case *build*
-and *host machines* are the same, but *target machine* is different.
+*build machine* is 64 bit OSX, your *host* machine is 32 bit ARM Linux
+and your *target machine* is irrelevant. This should be quite
```

Comment:
It's not really "irrelevant" as such. It is exactly the same as "host machine". Just leaving it hanging there may be confusing for newcomers. Maybe something like:

    your *host* machine is 32 bit ARM Linux and your *target machine*
    is irrelevant (but defaults to the same value as the *target machine*)

Also, for consistency it should be `*host machine*`, not `*host* machine`